ocs_ci/framework/conf/ocp_version/ocp-4.17-config.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# Config file for nightly OCP 4.17
+RUN:
+  client_version: "4.17.0-0.nightly"
+DEPLOYMENT:
+  installer_version: "4.17.0-0.nightly"
+  terraform_version: "1.0.11"
+  # ignition_version can be found here
+  # https://docs.openshift.com/container-platform/4.7/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks
+  ignition_version: "3.2.0"
+ENV_DATA:
+  # TODO: replace with 4.17 once template is available
+  vm_template: "rhcos-416.94.202403071059-0-vmware.x86_64"
+  acm_hub_channel: release-2.10
+  acm_version: "2.10"
```

Comment:
should be 2.11 for now and 2.11 once we have stable builds

Reply:
done